src/3d/framegraph/qgsambientocclusionrenderview.cpp

```diff
@@ -0,0 +1,183 @@
+/***************************************************************************
+  qgsambientocclusionrenderview.cpp
+  --------------------------------------
+  Date                 : June 2024
```

Comment:
June 2024 ? is that correct?

Reply:
yeah... too many thing to do! I will update it